assets/sass/modules/_forms.scss

```diff
@@ -247,3 +247,26 @@
 .chzn-container-active.chzn-with-drop .chzn-single div b {
   background-position: -17px 7px;
 }
+
+.custom-checkbox {
+  label {
+    background-image: url('/img/ui/forms/checkbox.png');
```

Comment:
nit: I think we should make it a little more obvious that this is a widget specific file, just in case (for whatever reason) someone creates a checkbox.png in courage and it gets overwritten in a future update. Probably something like checkbox-sign-in-widget.png is fine.